scripts/lib/libvirt/templates/cpu-intel.xml

```diff
@@ -4,7 +4,7 @@
     <pae/>
   </features>
   <cpu mode='custom' match='exact'>
-    <model fallback='allow'>core2duo</model>
+    <model fallback='allow'>Haswell-noTSX</model>
```

Comment:
Does this work on hosts with intel CPUs older than Haswell?

Reply:
my understanding is that fallback=allow allows to pick a different cpu that provides the "vmx" capability. from the documentation:

> If a hypervisor is not able to use the exact CPU model, libvirt automatically falls back to a closest model supported by the hypervisor while maintaining the list of CPU features